### ISSUE TITLE
Add Map Layer with Infrastructure Data

### ISSUE THREAD

**Issue by csc5k** [onaio/gisida#242]
We should add a map layer that shows infrastructure data. Currently it comes from two forms, one of which we should display everything from and one of which we should filter. I don't know the forms or how they should be filtered, but I will add that info when I receive it from the ROH team

**Comment by cKellyDesign** [comment]
@csc5k Yeah this will depend greatly on the forms I guess, so it will be interesting to see what come from that.

Another thing to keep in mind is that we can also expose different basemaps Styles from Mapbox, which can include quite a bit of infrastructure and other information (though i'm not sure how good it is in the refugee camps in Cox Bazaar...). Sounds like that would be supplemental if they're already using fancy forms to produce "infrastructure data", but something to keep in mind.
